src/types/student/student.ts

```diff
@@ -0,0 +1,24 @@
+export interface StudentInterface {
+    id: string;
+    token: string;
+    email: string;
+    status: string;
+    role: string;
+    recruiterId: object;
+    telephone: string;
+    firstName: string;
+    lastName: string;
+    githubUsername: string;
+    portfolioUrls?: string;
+    projectUrls: string;
+    bio: string;
+    expectedTypeWork: string;
+    targetWorkCity: string;
+    expectedContractType: string;
+    expectedSalary: string;
+    canTakeApprenticeship: boolean;
+    monthsOfCommercialExp: number;
+    education: string;
+    workExperience: string;
+    courses: string;
+}
```

Comment:
Should i add yarn.lock to commit?

Reply:
I think it's not necessary. 
It only write what commands was done in yarn, I think :)